test/tests/functional/pbs_multi_sched.py

```diff
@@ -580,6 +580,60 @@ def test_run_limts_per_scheduler(self):
         jc = "Not Running: User has reached server running job limit."
         self.server.expect(JOB, {'comment': jc}, id=jid4)
 
+    def test_list_multi_sched(self):
+        """
+        Test to verify that qmgr list sched works when multiple schedulers are present
+        """
+
+        self.setup_sc1()
+        self.setup_sc2()
+        self.setup_sc3()
+
+        self.server.manager(MGR_CMD_LIST, SCHED, expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc1", expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc2", expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc3", expect=True)
+
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="invalid_scname")
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
+        # delete sc3 sched
+        self.server.manager(MGR_CMD_DELETE, SCHED, id="sc3", sudo=True)
+
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="sc3")
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
+        self.server.manager(MGR_CMD_LIST, SCHED, expect=True)
+
```

Comment:
Same nit here, please remove the extra blank lines

Reply:
blank lines are used sparingly, since each manager call executes the targeted test objective, I consider each of them equivalent to a logical section.